webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -4,40 +4,50 @@ import store from '../javascripts/react_app/redux';
 
 import * as ToastActions from './react_app/redux/actions/toasts';
 
+const notificationTypeToAlertType = (type) => {
+  switch (type) {
```

Comment:
could we change the notice type to success already on the server?

object literals would be better instead of switch case ? (readability & efficient),